apps/paymaster-proxy/src/ProxyService.ts

```diff
@@ -16,7 +19,20 @@ export class ProxyService {
   static async init() {
     const redisClient = new Redis(envVars.REDIS_URL)
 
-    const apiServer = await initializeApiServer(redisClient)
+    const paymasterConfigs = [
+      getAlchemyPaymasterConfig({
+        chain: sepolia,
+        rpcUrl: envVars.ALCHEMY_RPC_URL_OP_SEPOLIA,
+        policyId: envVars.ALCHEMY_GAS_MANAGER_POLICY_ID_OP_SEPOLIA,
+      }),
+      getAlchemyPaymasterConfig({
+        chain: optimismSepolia,
+        rpcUrl: envVars.ALCHEMY_RPC_URL_OP_SEPOLIA,
+        policyId: envVars.ALCHEMY_GAS_MANAGER_POLICY_ID_OP_SEPOLIA,
+      }),
+    ] as const satisfies PaymasterConfig[]
+
+    const apiServer = await initializeApiServer(redisClient, paymasterConfigs)
```

Comment:
it might be better to pass an object into this, i imagine we're going to add much more to this overtime like loggers, sentry, etc